stream-chat-android-ui-components/src/main/res/drawable/stream_ic_mute.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M5.889,16H2C1.7348,16 1.4804,15.8946 1.2929,15.7071C1.1054,15.5196 1,15.2652 1,15V9C1,8.7348 1.1054,8.4804 1.2929,8.2929C1.4804,8.1054 1.7348,8 2,8H5.889L11.183,3.668C11.2563,3.608 11.3451,3.57 11.4391,3.5584C11.5331,3.5469 11.6284,3.5623 11.714,3.6028C11.7996,3.6434 11.872,3.7074 11.9226,3.7875C11.9732,3.8675 12.0001,3.9603 12,4.055V19.945C12.0001,20.0397 11.9732,20.1325 11.9226,20.2126C11.872,20.2926 11.7996,20.3567 11.714,20.3972C11.6284,20.4377 11.5331,20.4531 11.4391,20.4416C11.3451,20.4301 11.2563,20.3921 11.183,20.332L5.89,16H5.889ZM10,7.22L6.603,10H3V14H6.603L10,16.78V7.22ZM23.95,15.536L20.414,12L23.95,8.464L22.536,7.05L19,10.586L15.464,7.05L14.05,8.464L17.586,12L14.05,15.536L15.464,16.95L19,13.414L22.536,16.95L23.95,15.536Z"
+      android:fillColor="#808080"
```

Comment:
As Marton suggested we set fill color to black for consistency and apply tint afterwards